add-on/chrome/bootstrap.js

```diff
@@ -313,6 +321,13 @@ var WindowListener = {
           return;
         }
 
+        // Load the frame script into any tabs, plus any that get created.
+        // Note: We'd ideally do this only when tab sharing, but as we can't
```

Comment:
I think it's unnecessary to put this comment here... the way loading frame scripts work is not going to change in the future, so this sounds a bit like reminiscing :wink: